src/routes/console/project-[project]/storage/bucket-[bucket]/header.svelte

```diff
@@ -34,7 +34,7 @@
         <CoverTitle href={`/console/project-${projectId}/storage`}>
             {$bucket?.name}
         </CoverTitle>
-        <Id value={$bucket?.$id} event="bucket">Bucket ID</Id>
+        <Id value={$bucket?.$id} event="bucket">{$bucket?.$id ?? 'Bucket ID'}</Id>
```

Comment:
@TGlide  I think we just want to show the bucket id instead of the text

Reply:
That's what's going on here, but what if the ID is undefined? I could hide the ID instead

Reply:
When is it undefined? The page depends on the bucket in the layout so there's always an ID right? :thinking: